cfg/config.go

```diff
@@ -67,18 +67,18 @@ type SnapshotConfiguration struct {
 }
 
 type NATSConfiguration struct {
-	URLs                    []string `toml:"urls"`
-	SubjectPrefix           string   `toml:"subject_prefix"`
-	StreamPrefix            string   `toml:"stream_prefix"`
-	ServerConfigFile        string   `toml:"server_config"`
-	SeedFile                string   `toml:"seed_file"`
-	CredsUser               string   `toml:"user_name"`
-	CredsPassword           string   `toml:"user_password"`
-	CAFile                  string   `toml:"ca_file"`
-	CertFile                string   `toml:"cert_file"`
-	KeyFile                 string   `toml:"key_file"`
-	BindAddress             string   `toml:"bind_address"`
-	ConnectTotalWaitMinutes int      `toml:"connect_total_wait_minutes"`
```

Comment:
You need to assign a default value to this one, may be a default value of 5 will be good enought. Checkout where I initialize BindAddress and other values.